integration/sawtooth_integration/tests/test_dynamic_network.py

```diff
@@ -205,16 +205,12 @@ def stop_node(self, num):
 
     # if the validators aren't in consensus, wait and try again
     def assert_consensus(self):
-        sleep_time = 3
-        growth_rate = 2
-        for _ in range(5):
-            if self.in_consensus():
-                return
-
-            time.sleep(sleep_time)
-            sleep_time *= growth_rate
+        timeout = ASSERT_CONSENSUS_TIMEOUT + time.time()
+        in_consensus = False
 
-        self.assertTrue(self.in_consensus())
+        while not in_consensus and time.time() < timeout:
+            in_consensus = self.in_consensus()
```

Comment:
`in_consensus` includes a call to `sawtooth block list`. If the validators aren't in consensus, how many times will `in_consensus` be called, and how frequently?

Reply:
After running the tests a few times, it looks like in_consensus is called 1 to 6 times per call to assert_consensus. It doesn't seem to be affecting test stability, but we could add some sleeps back in to throttle it, if that's what you're worried about.